webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -258,19 +259,32 @@ object ExpectedReportsSerialisation {
     )
   }
 
+
+  def jsonComponentExpectedReport(c : ExpectedValue): JValue = {
+    c match {
+      case v : ExpectedValueId =>
```

Comment:
we will need to still be able to deserialize old data structure to be able to have compliance for old explect reports